src/main/webapp/WEB-INF/jsp/courseConfiguration.jsp

```diff
@@ -218,17 +231,35 @@
             }
         });
 
-        function confirmChoice(msg, button, callback) {
+        $('#conversionConfirm').change(function(){
+            if (this.checked) {
+                $('#pushGradesToCanvas').removeClass('hidden');
+                $('#conversionConfig').removeClass('hidden');
+            } else {
+                $('#pushGradesToCanvas').addClass('hidden');
+                $('#conversionConfig').addClass('hidden');
+                if(hasAssignmentConfiguration() == true) {
+                    confirmChoice('Turning off the grading feature will delete the Attendance Assignment from Canvas. Do you want to continue?', 'Delete Assignment Confirmation');
+                }
+            }
+        });
+
+        function hidePushingAlert (){
+            $('#pushConfirmation').modal('hide');
+        }
+
+        function confirmChoice(msg, title) {
             $.confirm({
                 text: msg,
-                title: button,
+                title: title,
                 cancelButton: "No",
                 confirm: function() {
-                    callback();
+                    var form = document.getElementById('sectionSelect');
```

Comment:
Any reason why this is using raw javascript when the rest of the file and application use jquery to do selections?